python/python_fastapi_server/routers/edr.py

```diff
@@ -440,28 +475,82 @@ async def get_collectioninfo_for_id(
 
     return collection, ttl
 
+def get_param_el(edr_collection_name:str, param_id:str):
+    """Gets the EDRParameter configuration based on the edr collection name and parameter id
+
+    Args:
+        edr_collection_name (str): edr collection name
+        param_id (str): parameter id
+
+    Returns:
+        _type_: parameter element_
+    """
+    edr_collections = get_edr_collections()
+    edr_collection_parameters= edr_collections[edr_collection_name]["parameters"]
+    for param_el in edr_collection_parameters:
+        if param_id == param_el["name"]:
+            return param_el
+    return None
+
+def get_param_metadata( param_id:str, edr_collection_name:str,wmslayers:dict=None)->dict:
```

Comment:
This functions is sometimes called with `wmslayers` (in metadata endpoints, I think), and sometimes without (in data endpoints?). This means the Parameters in both can/will be different. Is that what we want?

Reply:
For now I removed the description. Otherwise we need to fetch the getcapabilities for everything. Not sure if we really want the description at this stage.